FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessaging.h

```diff
@@ -146,7 +146,7 @@ NS_SWIFT_NAME(MessagingDelegate)
 ///
 /// * Subscribing to any topics.
 - (void)messaging:(FIRMessaging *)messaging
-    didReceiveRegistrationToken:(NSString *)fcmToken
+    didReceiveRegistrationToken:(NSString *__nullable)fcmToken
```

Comment:
nit: Use `nullable NSString *` (newer, more concise) over `NSString *__nullable`.

Reply:
@ryanwilson Need your approval since this is a public API change

Reply:
+1 - Morgan's change makes sense to me and this LGTM

Reply:
Done!